modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,123 @@
+
+* With this update, Argo CD CLI has been enhanced by enabling the `--app-namespace` flag for various application-related commands. link:https://issues.redhat.com/browse/GITOPS-3834[GITOPS-3834]
++
+This flag allows users to specify the namespace for operations such as waiting, rolling back, patching, editing, diffing, viewing history, syncing, and deleting applications. As a result, the following commands are updated in {gitops-title}:
++
+** `argocd app wait`
+** `argocd app rollback`
+** `argocd app patch`
+** `argocd app edit`
+** `argocd app diff`
+** `argocd app history`
+** `argocd app sync`
+** `argocd app delete`
+** `argocd app get`
+
+[id="fixed-issues-1-13-0_{context}"]
+== Fixed issues
+
+* Before this update, if the `DeploymentConfig [apps.openshift.io/v1]` API was disabled in the {OCP} cluster, the {gitops-shortname} Operator would enter the `CrashLoopBackoff` error state and would not function properly. This update fixes the issue by modifying the functionality of the {gitops-shortname} Operator to ensure that it does not enter the `CrashLoopBackoff` error state when the API is unavailable in the cluster. link:https://issues.redhat.com/browse/GITOPS-4489[GITOPS-4489]
+
+* Before this update, users were unable to connect to remote Git repositories through a SOCKS5 proxy server because it was only compatible with HTTPS URLs. With this update, users can access remote Git repositories by using a SOCKS5 proxy server that supports the SSH protocol. link:https://issues.redhat.com/browse/GITOPS-3710[GITOPS-3710]
+
+* Before this update, the `forceHttpBasicAuth` repository configuration was not initialized properly during repository updates from Argo CD UI. As a result, the value was reset to default, leading to connection errors, particularly for Azure DevOps repositories. This update fixes the issue by adding a change in the upstream to ensure that the `forceHttpBasicAuth` configuration is correctly initialized during repository update operations. link:https://issues.redhat.com/browse/GITOPS-3706[GITOPS-3706]
+
+* Before this update, the Argo CD CLI would designate the initial source as the main source for multi-source applications. With this update, the Argo CD CLI supports multi-source applications by considering all sources during processing. link:https://issues.redhat.com/browse/GITOPS-2623[GITOPS-2623]
+
+* Before this update, the RolloutManager controller did not create ServiceMonitor. With this update, the RolloutManager controller is improved to include support for ServiceMonitor creation. As a result, you can gather metrics from Rollouts using the enhanced RolloutManager controller, and these metrics are visible in monitoring. link:https://issues.redhat.com/browse/GITOPS-3271[GITOPS-3271]
```

Comment:
@Dhruv-Soni11 It is either the Argo Rollouts controller CR or RolloutManager custom resource (CR). CHeck what exactly to use in this case. There is no such thing to use as "RolloutManager controller".

Also please check on ServiceMonitor. Must be mentioned in within backticks. Also, identify what API object it is and mention it here. Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting.

Reply:
Fixed as Argo Rollouts controller CR.